pkg/optjson/optjson.go

```diff
@@ -53,3 +53,41 @@ func (s *String) UnmarshalJSON(data []byte) error {
 	s.Valid = true
 	return nil
 }
+
+type Slice[T any] struct {
```

Comment:
Hello generics!

Reply:
probably the first one, right? I can also make this a `type StringSlice` if we want to avoid generics, happy to adjust @gillespi314 and @mna just let me know